apps/framework-docs/llm-docs/python/table-setup.md

```
@@ -258,6 +328,12 @@ time_series_table = OlapTable[TimeSeriesData](
    - Optimize table settings
    - Back up important data
 
+5. **External Tables**
+   - Mark CDC/legacy tables as EXTERNALLY_MANAGED
+   - Configure dev mirrors for MaterializedView development
+   - Keep schemas in sync with `moose db pull`
+   - Use fallback mode for developers without production access
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)